diff --git a/main/gcc/APKBUILD b/main/gcc/APKBUILD
index fe52d00a..a64835cc 100644
--- a/main/gcc/APKBUILD
+++ b/main/gcc/APKBUILD
@@ -14,7 +14,7 @@ license="GPL-2.0-or-later LGPL-2.1-or-later"
 _gccrel=$pkgver-r$pkgrel
 depends="binutils$_target isl"
 makedepends_build="gcc$_cross g++$_cross paxmark bison flex texinfo gawk zip gmp-dev mpfr-dev mpc1-dev zlib-dev"
-makedepends_host="linux-headers gmp-dev mpfr-dev mpc1-dev isl-dev zlib-dev !gettext-dev $pkgname-gnat-bootstrap"
+makedepends_host="linux-headers gmp-dev mpfr-dev mpc1-dev isl-dev zlib-dev !gettext-dev" # $pkgname-gnat-bootstrap"
 subpackages=" "
 [ "$CHOST" = "$CTARGET" ] && subpackages="gcc-doc$_target"
 replaces="libstdc++ binutils"

